floodwatch/src/js/components/FindInDemographics.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+import DemographicKeys from '../../stubbed_data/demographic_keys.json';
+import type {PersonResponse} from '../api/types';
+import type {Preset, Filter, DisabledCheck} from './filtertypes'
+
+export function getCategoryKey(category: string): string {
```

Comment:
Oh perfect! Guess I forgot to pull. Let's forget I asked for the sake of my pride :D